sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/DigitalGyro.scala

```diff
@@ -55,4 +53,36 @@ abstract class DigitalGyro(tickPeriod: Time) {
       retrieveVelocity - currentDrift
     }
   }
+
+  def checkCalibration: Boolean = {
+    var successful = true
+    val chunks: List[List[Value3D[AngularVelocity]]] = {
+      val part1 = calibrationVelocities.splitAt(calibrationVelocities.size / 5)
+      val part2 = part1._2.splitAt(calibrationVelocities.size / 5)
+      val part3 = part2._2.splitAt(calibrationVelocities.size / 5)
+      val part4 = part3._2.splitAt(calibrationVelocities.size / 5)
+      List(part1._1.toList, part2._1.toList, part3._1.toList, part4._1.toList, part4._2.toList)
+    }
+    println("Checking Calibration")
+    chunks.foreach({chunk: List[Value3D[AngularVelocity]] =>
+      val sumChunk = chunk.reduceLeft{ (acc, cur) =>
+        acc + cur
+      }
+      val average = sumChunk * (1D / chunk.size)
+      println(s"Chunk Average: $average")
+      if((average.x - currentDrift.x).abs > threshold.x){
+        println(s"Drift is not within x threshold")
+        successful = false
+      }
+      if((average.y - currentDrift.y).abs > threshold.y){
+        println(s"Drift is not within y threshold")
+        successful = false
+      }
+      if((average.z - currentDrift.z).abs > threshold.z){
+        println(s"Drift is not within z threshold")
+        successful = false
+      }
+    })
+    successful
```

Comment:
Also check that the queue of calibration values is full

Reply:
On that note, please make the queue size 1,000 values instead if 200. This means that we'll use 5 seconds worth of data instead of 1 second (at 5ms update loops)